dashboard/app/controllers/aichat_controller.rb

```diff
@@ -152,7 +152,9 @@ def student_chat_history
       return render(status: :forbidden, json: {error: "Access denied for student chat history."})
     end
 
-    aichat_events = AichatEvent.where(user_id: student_user_id, level_id: level_id, script_id: script_id).order(created_at: :desc).pluck(:aichat_event)
+    aichat_events = AichatEvent.where(user_id: student_user_id, level_id: level_id, script_id: script_id).order(:created_at).pluck(:aichat_event).map do |event|
```

Comment:
Want the oldest event to be first, not last since student chat history lists events from oldest to most recent.